crux/src/Crux.hs

```diff
@@ -205,6 +205,7 @@ showVersion nm ver = sayCrux (Log.Version nm ver)
 --
 -- The following CruxOptions affect the generated OutputConfig:
 --
+--  * noColors       (default stance: False)
```

Comment:
This default stance note needs to change now that we are using `hSupportsANSIColor`

Reply:
Fixed!